src/navigation/pagination/nextPage/styles.ts

```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components';
+
+import SVGIcon from '../arrow';
+
+import colors from 'styles/colors';
+
+import { WrapperType } from './types';
+
+export const Wrapper = styled.button<WrapperType>`
```

Comment:
como esse componente é o mesmo usado em next and previous page, podemos centralizar em um local só e utilizar o mesmo, sugiro que seja criado em `pages/styles.ts`